source/_includes/dashboard/edit_dashboard.md

```diff
@@ -0,0 +1,7 @@
+{% capture name %}{{ include.name | default: page.title }}{% endcapture %}
+{% capture domain %}{{ include.domain | default: page.ha_domain }}{% endcapture %}
+{% capture title %}{{ include.title | default: page.title }}{% endcapture %}
+
+To add the {{ title | downcase }} to your user interface:
+1. In the top right of the screen, select the three dots menu and then select **Edit dashboard**. 
```

Comment:
Should we add a sub-step here about the dialog you get the first time you do this?
![image](https://github.com/home-assistant/home-assistant.io/assets/15093472/f286890b-245a-49b9-8af9-31d1bc15236f)